{"uuid":"e0e0ea32-a25e-48bf-8f2d-c662147ddeea","name":"Scenario Outline: To validate if user getting error message when enter valid python code in operations stack page","children":["540fd468-0e99-40cd-bded-e73abd5f38b6"],"befores":[],"afters":[{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055353521,"stop":1709055353523},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055354178,"stop":1709055354179},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055356301,"stop":1709055356301},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055356958,"stop":1709055356959},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055357375,"stop":1709055357376},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055358270,"stop":1709055358270},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055358976,"stop":1709055358977},{"name":"stepDefinitions.ApplicationHooks.tearDown()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709055358977,"stop":1709055360442}],"start":1709055347348,"stop":1709055360442}